templates/16-04/index.html

```diff
@@ -211,10 +211,10 @@ <h4>
         Infrastructure packages
       </h4>
       <p>
-        Canonical will provide extended security maintenance to all binary packages that reside in Ubuntu Main Repository.
+        Canonical provides extended security maintenance to all binary packages that reside in Ubuntu Main Repository.
```

Comment:
https://docs.google.com/document/d/1wclofqY9b3v_M9igpkYn6or5uXx0B_Xb_nVVzGBbOLY/edit#heading=h.tbcblwt3kct9

```suggestion
        For Ubuntu 16.04 LTS, where technically feasible, Canonical provides extended security maintenance to all binary packages that reside in the Ubuntu Main Repository.
```